docs/changehistory/Roadmap.md

```diff
@@ -21,123 +21,129 @@ Volunteers for help on any or all of these projects are welcome. In particular,
+- [ ] Create examples and templates for building and publishing Extensions.
+- [ ] Support for better testing in the Extension's Hosting App, i.e. iTwin Viewer, iTwin Design Review, PlantSight, etc.
 
 ### GIS Display
 
-- Expand base map support to include additional sources (AzureMaps, MapBox etc.) or solid color. Transparency of base map can be controlled independently of layers.
-- Add support for display of raster map layers connected to live WMS, WMTS, AzureMaps, MapBox, or ArcGIS REST API servers. Map layer visibility and transparency can be controlled independently. Layers can be attached either as background below BIM geometry or overlays above BIM geometry.
+- [X] Expand base map support to include additional sources (AzureMaps, MapBox etc.) or solid color. Transparency of base map can be controlled independently of layers.
+- [X] Add support for display of raster map layers connected to live WMS, WMTS, AzureMaps, MapBox, or ArcGIS REST API servers. Map layer visibility and transparency can be controlled independently. Layers can be attached either as background below BIM geometry or overlays above BIM geometry.
 
 ### Microsoft Teams integration
 
-- Provide api for hosting iTwin.js applications directly within Microsoft Teams.
+- [ ] Provide api for hosting iTwin.js applications directly within Microsoft Teams.
 
 ### ET/IT/OT Integration
 
-- Provide working examples and templates for interfacing between engineering content in iModels, enterprise systems, and realtime IOT sensors, cameras, controllers, devices, and processors.
-- Create tools for augmenting data synchronized with connectors from engineering design tools with IOT-link elements.
-- Enhance BIS schemas for common IOT patterns and query engines.
+- [ ] Provide working examples and templates for interfacing between engineering content in iModels, enterprise systems, and realtime IOT sensors, cameras, controllers, devices, and processors.
+- [ ] Create tools for augmenting data synchronized with connectors from engineering design tools with IOT-link elements.
+- [ ] Enhance BIS schemas for common IOT patterns and query engines.
 
 ### Agent Deployment
 
-- Provide working examples and templates of agents deployed via Kubernetes.
-- Perform operations on iModels that are triggered by relevant events from iModelHub.
+- [ ] Provide working examples and templates of agents deployed via Kubernetes.
+- [ ] Perform operations on iModels that are triggered by relevant events from iModelHub.
 
 ### iTwin Viewer
 
-- Create fully functional iTwin web viewing application platform, deployable on any cloud infrastructure. The Bentley iTwin Design Review product will be built from this application.
-- Provide necessary infrastructure to host Extensions within any iTwin Viewer-based solution.
+- [ ] Create fully functional iTwin web viewing application platform, deployable on any cloud infrastructure. The Bentley iTwin Design Review product will be built from this application.
```

Comment:
@kckst8

Reply:
+ @Carl-Hinkle @calebmshafer 
We should probably split this into two, the itwin viewer is avaialble but DR hasnt moved to it yet

Reply:
@aruniverse nah, I'll remove the reference to Design Review. This is the iTwin.js roadmap, it doesn't track what applications are doing with iTwin.js.